global_helpers/panther_lookuptable_helpers.py

```diff
@@ -14,3 +15,16 @@ def _lookup(self, match_field: str, *keys) -> list or str:
         if isinstance(match, list):
             return [deep_get(match_value, *keys) if match_value else None for match_value in match]
         return deep_get(match, *keys)
+
+    def enrichments_by_pmatch(self, event, p_match: str) -> dict:
```

Comment:
`enrichments_by_pmatch` is descriptive but long. Is there a shorter name that is still conveys the action?

Reply:
yep, now called `p_matches`